service/history/api/resetworkflow/api.go

```diff
@@ -192,6 +192,7 @@ func GetResetReapplyExcludeTypes(
 	case enumspb.RESET_REAPPLY_TYPE_NONE:
 		exclude[enumspb.RESET_REAPPLY_EXCLUDE_TYPE_SIGNAL] = true
 		exclude[enumspb.RESET_REAPPLY_EXCLUDE_TYPE_UPDATE] = true
+	default:
```

Comment:
I just want to ask before approving: do we want to handle `RESET_REAPPLY_TYPE_UNSPECIFIED` and `RESET_REAPPLY_TYPE_ALL_ELIGIBLE` in the same way?

Perhaps it's worth a comment so that future readers understand that there is reasoning behind using `default:` to appease the linter

Reply:
No I confirmed with @dandavison, no need to handle other cases.

Reply:
added the cases explicitly.